src/main/java/org/mongojack/DBCursor.java

```diff
@@ -311,7 +311,9 @@ public int getOptions() {
      * the database
      * 
      * @return The number of get mores
+     * @deprecated will be removed in a future release
```

Comment:
for all these various deprecations can we copy the message from the thing we're wrapping instead of just saying "will be removed in a future release"? that will make it much easier for the users of mongojack to figure out how to deal with the deprecation warnings